content/200-orm/500-reference/050-prisma-client-reference.mdx

```diff
@@ -1889,6 +1868,123 @@ const usersWithCount = await prisma.user.findMany({
 </cmdResult>
 </CodeWithResult>
 
+### `omit` (Preview)
+
+`omit` defines which fields are excluded in the object that Prisma Client returns.
+
+#### Remarks
+
+- You cannot combine `omit` and `select` since they serve opposite purposes
+
+#### Examples
+
+##### Omit the `password` field from all User records
+
+<CodeWithResult>
+<cmd>
+
+```ts
+const result = await prisma.user.findMany({
+  omit: {
+    password: true,
+  },
+})
+```
+
+</cmd>
+<cmdResult>
+
+```js no-copy
+[
+  {
+    id: 1,
+    email: 'jenny@prisma.io',
+    name: 'Jenny',
+  },
+  {
+    id: 2,
+    email: 'rose@prisma.io',
+    name: 'Rose',
+  },
+]
+```
+
+</cmdResult>
+</CodeWithResult>
+
+##### Omit the `title` fields from all Post relations
```

Comment:
Is calling this "Posts relations" even correct though?
It is the `Post` model, but aren't the relations named "posts"?

Reply:
Ya it's not referenced like there anywhere in the doc. We can change it to: 
```suggestion
##### Omit the `title` fields from all User's posts
```